tests/bin/eslint.js

```diff
@@ -141,10 +141,14 @@ describe("bin/eslint.js", () => {
     });
 
     describe("running on files", () => {
-        it("has exit code 0 if no linting errors occur", () => assertExitCode(runESLint(["bin/eslint.js"]), 0));
-        it("has exit code 0 if a linting warning is reported", () => assertExitCode(runESLint(["bin/eslint.js", "--env", "es6", "--no-eslintrc", "--rule", "semi: [1, never]"]), 0));
-        it("has exit code 1 if a linting error is reported", () => assertExitCode(runESLint(["bin/eslint.js", "--env", "es6", "--no-eslintrc", "--rule", "semi: [2, never]"]), 1));
-        it("has exit code 1 if a syntax error is thrown", () => assertExitCode(runESLint(["README.md"]), 1));
+        it("has exit code 0 if no linting errors occur", () =>
+            assertExitCode(runESLint(["bin/eslint.js", "--rulesdir=tools/internal-rules"]), 0));
+        it("has exit code 0 if a linting warning is reported", () =>
+            assertExitCode(runESLint(["bin/eslint.js", "--env", "es6", "--no-eslintrc", "--rule", "semi: [1, never]"]), 0));
+        it("has exit code 1 if a linting error is reported", () =>
+            assertExitCode(runESLint(["bin/eslint.js", "--env", "es6", "--no-eslintrc", "--rule", "semi: [2, never]"]), 1));
+        it("has exit code 1 if a syntax error is thrown", () =>
+            assertExitCode(runESLint(["README.md", "--rulesdir=tools/internal-rules"]), 1));
```

Comment:
Why did you change these lines?

Reply:
These test cases rely on ESLint linting itself. I added `--rulesdir=tools/internal-rules` to the second test case to ensure that the config file is valid for that run. The goal is to ensure that the exit code is 1 as a result of the syntax error, not as a result of the missing rule.